Samples/Controllers/NorthwindController.cs

```diff
@@ -137,10 +126,8 @@ orderby o.OrderDate
 
         [HttpGet]
         public object SalesByCategory() {
-            // NOTE see the #warning at the top of the file
-            var temp = _nwind.Order_Details.Include(d => d.Product.Category).ToArray();
-
-            return from d in temp
+#warning TODO remove ToArray() when https://github.com/aspnet/EntityFramework/issues/3676 fix is released
+            return from d in _nwind.Order_Details.Include(d => d.Product.Category).ToArray()
```

Comment:
It is not clear for me form [aspnet/EntityFramework#3676](https://github.com/aspnet/EntityFramework/issues/3676l)  why two LINQ queries using `let` keyword and `group by` method works differently. So, if it does not works without ` .ToArray()` let's just trust you and hope that it will be fixed in EF 1.0.1 ;)